_about/notes/index.md

```diff
@@ -16,18 +16,20 @@ toc: false
 
 {% include section-index.html docs=site.about %}
 
-The [latest Istio monthly release](https://github.com/istio/istio/releases) is {{site.data.istio.version}} which
-you can download with:
+- The [latest Istio monthly release](https://github.com/istio/istio/releases) is {{site.data.istio.version}} which
```

Comment:
There shouldn't be a - here and in the next item, it's not necessary and doesn't mesh well with the bulleted list of releases that are displayed right above.

Reply:
k I didn't notice the inlined TOC there, let me update locally and fix it up
(the issue is the doc link and the curl examples look detached from their respective release)

Reply:
@geeknoid why do we need to list the releases twice - it's already on the left bar ?

also the period at the end of "Istio releases information." looks odd, would be a ":" instead for that toc to work

how about:
![screen shot 2018-03-28 at 3 40 09 pm](https://user-images.githubusercontent.com/3664595/38060151-59d451d4-329e-11e8-93ca-85f476e54a4d.png)